lib/ApplicationServer.js

```diff
@@ -139,6 +139,14 @@ ApplicationServer.prototype.run = function() {
     }
   }
 
+  if (this.config_['server']['cors'] && this.config_['server']['cors']['enabled']) {
```

Comment:
seems that bohr has a dependency on lodash package. Would make more sense to use [_.get()](https://lodash.com/docs/4.17.2#get) to check thath cors is enabled?